README.md

```diff
@@ -39,6 +39,11 @@ export const routes: Routes = [
 ];
 ```
 
+**NOTE**: When specifying a relative path to lazy loaded module, one of the following two conditions *must* hold:
+
+* The lazy loaded route is defined in the same module file where it is added by `RouterModule.forXxx()`
```

Comment:
One way or the other, the routes will be "included" in the module file. The idea was to clarify that you either put them directly in module file, or you can put them in another file, which has to be a sibling of module file. In both cases, the path must be relative to the file.